pydantic/fields.py

```diff
@@ -106,6 +108,8 @@ class FieldInfo(_repr.Representation):
         validation_alias: The validation alias of the field.
         serialization_alias: The serialization alias of the field.
         title: The title of the field.
+        title_priority: Priority of the field's title. This affects whether a title generator is used.
```

Comment:
Are you sure we need this? If so, I think it needs more documentation.

Reply:
I guess I wanted to have a similar API to `alias_generator` and `alias_priority`.